.gitmodules

```diff
@@ -27,4 +27,4 @@
 	branch = zeus
 [submodule "layers/meta-rust-bin"]
 	path = layers/meta-rust-bin
-	url = https://github.com/rust-embedded/meta-rust-bin.git
+	url = https://github.com/3mdeb/meta-rust-bin.git
```

Comment:
This can be changed back to rust-embedded as soon as this gets merged: https://github.com/rust-embedded/meta-rust-bin/pull/71

Reply:
It looks like the other submodules are still at the zeus branch. Is that correct?

Reply:
@brocaar Hmm, I guess I'm not sure what is the impact of this field there. When I clone the repo and update the modules (`make submodules`) they all do have correct revisions (from dunfell, not from zeus).

Reply:
I can update the branches in the `.gitmodules` file as well.